crates/bevy_solari/src/scene/blas.rs

```diff
@@ -14,13 +14,21 @@ use bevy_render::{
     render_resource::*,
     renderer::{RenderDevice, RenderQueue},
 };
+use std::collections::VecDeque;
+
+/// After compacting this many vertices worth of meshes per frame, no further BLAS will be compacted.
+/// Lower this number to distribute the work across more frames.
+const MAX_COMPACTION_VERTICES_PER_FRAME: u32 = 400_000;
```

Comment:
Could this be a plugin setting or resource?

Reply:
Eventually yes, for now nothing in solari is configurable. It's in the tracking issue.